var/spack/repos/jcsda-emc/packages/gsw/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Gsw(CMakePackage):
+    """Gibbs-SeaWater (GSW) Oceanographic Toolbox in Fortran"""
+
+    homepage = "https://github.com/JCSDA/GSW-fortran"
+    git = "https://github.com/JCSDA/GSW-fortran.git"
+
+    maintainers = ["climbfuji"]
+
+    depends_on("ecbuild", type=("build"))
+    depends_on("ecbuild@3.3.2:", type=("build"), when="@3.0.7:")
+
+    version("3.0.7", commit="1a02ebaf6f7a4e9f2c2d2dd973fb050e697bcc74")
```

Comment:
Note the non-standard notation of the tag. In the repo, it's [v3.07](https://github.com/JCSDA-internal/GSW-Fortran/releases/tag/v3.07)